providers/implementations/kdfs/x942kdf.c

```diff
@@ -170,7 +170,7 @@ static int der_encode_sharedinfo(WPACKET *pkt, unsigned char *buf, size_t buflen
  * |cek_oidlen| The length (in bytes) of the key wrapping algorithm oid,
  * |acvp| is the optional blob of DER data representing one or more of the
  *   OtherInfo fields related to |partyu|, |partyv|, |supp_pub| and |supp_priv|.
- *   This field should noramlly be NULL. If |acvp| is non NULL then |partyu|,
+ *   This field should normally be NULL. If |acvp| is non NULL then |partyu|,
```

Comment:
This change is good.  The Shakespeare quotes are from the [earliest version](https://en.wikipedia.org/wiki/To_be,_or_not_to_be#Second_Quarto_(1604)) of the play are don't need to be correct (since they aren't wrong).

Reply:
@cuishuang I am not sure whether you misunderstood @paulidale or whether you are playing a (one day late) april's fool joke with us? You just reverted the only change which would have been valid and left only the "corrections" of the Hamlet quotations which are in their correct historical writing and shouldn't be changed.

Reply:
Sorry, I misunderstood what @paulidale  meant ￣□￣